softwerkskammer/lib/mailsender/mailsenderService.js

```diff
@@ -19,7 +19,7 @@ const misc = beans.get('misc');
 const mailtransport = beans.get('mailtransport');
 
 function sendMail(message, type, callback) {
-  mailtransport.sendMail(message, type, conf.get('sender-address'), callback);
+  mailtransport.sendMail(message, type, conf.get('sender-address'), conf.get('include-footer'), callback);
```

Comment:
Die Option in der json heißt "include-mail-footer", hier wird aber "include-footer" abgefragt. Das passt nicht zusammen, oder uebersehe ich da was?

Reply:
Wurde nach dem pull gefixt